catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -259,6 +259,12 @@
         <reference-listener ref="workspaceTransformations" bind-method="bindPlugin" unbind-method="unbindPlugin"/>
     </reference-list>
 
+    <bean id="queryResponseTransformer" class="ddf.catalog.util.impl.SortedServiceReferenceList"/>
+    <reference-list interface="ddf.catalog.transform.QueryResponseTransformer"
+                    member-type="service-reference">
```

Comment:
You will probably want to add a filter on this reference-list for a new property on query response transformers that should be available for table export.